client/app.jsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
-import Home from './pages/home';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import SpotifyLogin from './pages/spotify-login';
+import SpotifyAuthRedirect from './pages/spotify-auth-redirect';
 
 export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // route: parseRoute(window.location.hash)
+    };
+  }
```

Comment:
You can remove this constructor for now. Since you are using React Router, you won't be needing any of the hash routing stuff.